### PR TITLE
r.resamp.filter: update manual

### LINKED ISSUE #2569: [Feat] Provide better explanation for radius parameter for r.resamp.filter
I want to resample a raster from 15m to 460m using a Gaussian filter.

**The goal**

I am having a coarse image which I want to downscale. I also have a fine resolution band to assist the downscaling. The downscaling method I am using is called geographically weighted area-to-point regression Kriging (GWATPRK). The method consists of two steps:

1. GWR and,
2. ATPK on the GWR's residuals.

In order to perform GWR using raster data, those needs to have the same pixel size. This means that, my fine resolution image needs to be upscaled to match the spatial resolution of the coarse band. This upscaling of the fine band needs to be done using a Gaussian kernel (i.e., the PSF). I have found that GRASS GIS has a tool called `r.resamp.filter`. I am trying to run the function but I am getting the following error(s):

1. `ERROR: Differing number of values for filter= and [xy_]radius=`

This error occurs when I use two filter kernels (e.g., `gauss + box`, or `gauss + bartlett`). I am using two kernels because according to the Manual:

_Kernels with infinite extent (Gauss, normal, sinc, Hann, Hamming, Blackman) must be used in conjunction with a finite windowing function (box, Bartlett, Hermite, Lanczos)._

Doesn't matter what numbers I put in the `Filter radius` or `Filter radius (horizontal)` and `Filter radius (vertical)` (see image below), I tested A LOT of numbers.
8JFqN

2. `ERROR: At least one filter must be finite`

This error occurs when I use one filter kernel, the gaussian (I am interested in applying a Gaussian filter, because I want to model the point spread function during downscaling satellite imagery).

The steps I followed were:

`r.external` to import the raster
`g.region` where I set the region using my original fine resolution image BUT in _Resolution_ tab I changed the 2D resolution into 460
`r.resamp.filter` and the errors I mentioned
Ultimately, I want to apply a Gaussian filter with sigma (std) = 0.5 to my image while I am upscaling it.

Here the [image](https://drive.google.com/drive/folders/18_1Kshb8WbT04gwOw4d_xhfQenULDXdB?usp=sharing) I am using

### PR DESCRIPTION
**Pull request by metzm**
Update the manual according to #2569 

Fixes #2569

### CODE REVIEW

**Comment by neteler** [comment]
> Isn't it overkill and an unnecessary waste of resources that all (also build) tests are started when a single manual is updated?

That was also my thought once: 

[Feat] CI: add support for [skip ci] #2174

**Comment by metzm** [comment]
> > Isn't it overkill and an unnecessary waste of resources that all (also build) tests are started when a single manual is updated?
> 
> That was also my thought once:
> 
> [Feat] CI: add support for [skip ci] #2174

Didn't know this, very helpful for cases like this: https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/ So "the contributor needs to carefully decide" it seems.

File: raster/r.resamp.filter/r.resamp.filter.html [hunk 1]
```diff
@@ -37,8 +43,33 @@ <h2>DESCRIPTION</h2>
...
+<p>
+When resampling a map to a lower resolution (alternative to
+aggregation, e.g. <a href="r.resamp.stats.html">r.resamp.stats</a>), a
+good starting point is to use the filters <b>gauss,box</b> with the
+radii <em>0.25 * output resolution</em>, <em>1 * output
+resolution</em>.
+
+<p>
+These are recommendations for initial settings, filters and radii might
+need adjustment according to the actual purpose.
```
Comment (echoix) [inline raster/r.resamp.filter/r.resamp.filter.html:72]: I've reread a couple of times to understand what was meant, the comma was kind of weird

**Comment by echoix** [comment]
> > > Isn't it overkill and an unnecessary waste of resources that all (also build) tests are started when a single manual is updated?
> 
> > 
> 
> > That was also my thought once:
> 
> > 
> 
> > [Feat] CI: add support for [skip ci] #2174
> 
> 
> 
> Didn't know this, very helpful for cases like this: https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/ So "the contributor needs to carefully decide" it seems.

Also, the final commit shouldn't be skipped, as some checks are required, and won't run. Unless the one merging has super-powers and can force it, but them he/she needs to make sure the squashed-merged commit isn't skipped so the CI on merge still runs

File: raster/r.resamp.filter/r.resamp.filter.html (hunk 1 above)
Comment (metzm) [reply raster/r.resamp.filter/r.resamp.filter.html]: Better now?

**Comment by metzm** [comment]
> > 
> > Didn't know this, very helpful for cases like this: https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/ So "the contributor needs to carefully decide" it seems.
> 
> Also, the final commit shouldn't be skipped, as some checks are required, and won't run. Unless the one merging has super-powers and can force it, but them he/she needs to make sure the squashed-merged commit isn't skipped so the CI on merge still runs

[skip ci] does not work as expected, I don't see a need to run the required build checks for changes in a manual. HTML syntax checking should be sufficient in this case? Maybe the issue #2174 should be re-opened?

File: raster/r.resamp.filter/r.resamp.filter.html (hunk 1 above)
Comment (echoix) [reply raster/r.resamp.filter/r.resamp.filter.html]: Yep!

**Review by echoix — APPROVED** [review]

**Pull request merged**